Gemfile

```
@@ -5,13 +5,13 @@ gemspec
 
 # Private runtime dependencies
 gem 'nokogiri', '~> 1.13', platform: :ruby
-gem 'sisjwt', '~> 0.3', github: 'tractionguest/sisjwt-ruby'
+gem 'sisjwt', github: 'tractionguest/sisjwt-ruby', branch: 'feature-kube-rails-7-upgrade'
```

change to
```suggestion
gem 'sisjwt', '~> 0.4.0', github: 'tractionguest/sisjwt-ruby'
```
when this [PR](https://github.com/tractionguest/sisjwt-ruby/pull/7) is merged